sources/Cargo.toml

```diff
@@ -65,15 +65,21 @@ members = [
 
     "models",
 
+    "settings-extensions/autoscaling",
     "settings-extensions/aws",
+    "settings-extensions/bootstrap-container",
```

Comment:
this should be "bootstrap-containers" for consistency with "host-containers"

Reply:
Sure, will change.